scripts/cnode-helper-scripts/blockPerf.sh

```diff
@@ -138,6 +154,142 @@ else
     echo $! > $pidfile
 fi
 
+missingTbh=true; missingCbf=true; 
+
+getDeltaMS() {
+  echo $(echo "$2 $4" | awk -F '[:, ]' '{print ($1*3600000+$2*60000+$3*1000+$4)-($5*3600000+$6*60000+$7*1000+$8) }' || 0)
+}
+
+getSlotDate() {
+  echo $(date -d @$(( $1 + $NETWORK_UTIME_OFFSET )) +'%F %T')
+}
+
+reportBlock() {
+  if [[ ! -z "${blockLog}" ]]; then
+    blockLogLineCycles=0
+    while IFS= read -r blockLogLine;do
+      # parse block and propagation metrics from different log kinds
+      lineKind=$(jq -r .data.kind <<< $blockLogLine)
+      case $lineKind in
+        ChainSyncClientEvent.TraceDownloadedHeader)
+          if $missingTbh; then
+            line_tsv=$(jq -r '[
+             .at //0,
+             .data.slot //0,
+             .data.block //0
```

Comment:
Is this needed? It looks like you don't reference line_data_arr[2]?

Reply:
Could be that its added because line_tsv is printed later in the script, and having the block in output might be desirable there?

Reply:
right, this is a leftover from an older version, where we parsed blockNum instead of just using EKGs most recent blockHeight

Reply:
removed `data.block`, and added `data.peer.remote.addr|port` to also parse the header announcing peer